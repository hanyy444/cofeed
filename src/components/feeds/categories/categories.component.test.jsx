import { render, screen } from "@testing-library/react";
import Categories from "./categories.component";

describe('Test categories component', () => { 
    expect.assertions(2)
    it('Should render without crashing', () => {
        render(<Categories/>)
        const categories = screen.getByTestId('categories')
        expect(categories).toBeInTheDocument()
        expect(categories).toMatchSnapshot()
    })
 })