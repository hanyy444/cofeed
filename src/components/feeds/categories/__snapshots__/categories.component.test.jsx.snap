// Vitest Snapshot v1

exports[`Test categories component > Should render without crashing 1`] = `
<ul
  class="categories"
  data-testid="categories"
>
  <li
    class="category"
  >
    all
  </li>
  <li
    class="category"
  >
    following
  </li>
  <li
    class="category"
  >
    popular
  </li>
  <li
    class="category"
  >
    newest
  </li>
</ul>
`;
